acmw-db-app/components/AddMeetingForm.jsx

```diff
@@ -0,0 +1,190 @@
+import styles from '../styles/components/Form.module.css'
+import DateSelectionForm from './FormComponents/DateSelectionForm'
+import SelectInput from './FormComponents/SelectInput'
+import SubmitButton from './FormComponents/SubmitButton'
+import TextField from './FormComponents/TextField'
+import TimeSelectionForm from './FormComponents/TimeSelectionForm'
+import {useEffect, useRef, useState} from 'react'
+import CompanySearchInput from './FormComponents/CompanySearchInput'
+import SubmitNotification from './FormComponents/SubmitNotification'
+
+const AddCompanyForm = (props) => {
+
+    const subscribed = useRef(false)
+    const meetingName = useRef("")
+    const semester = useRef("")
+    const date = useRef({
+        month: null,
+        day: null,
+        year: null,
+    })
+    const time = useRef({
+        hours: null,
+        minutes: null,
+        timeOfDay: null
+    })
+
+    const companyOptions = [{label: "No"}, {label: "Yes"}]
+    const [companyMeeting, setCompanyMeeting] = useState(false)
+    const company = useRef({
+        name: null,
+        id: null
+    })
+
+    const [meetingError, setMeetingError] = useState("")
+    const [semesterError, setSemesterError] = useState("")
+    const [dateError, setDateError] = useState("")
+    const [timeError, setTimeError] = useState("")
+    const [companyError, setCompanyError] = useState("")
+    const [code, setCode] = useState("")
+    const [showNotif, setShowNotif] = useState(false)
+
+    useEffect(() => {return () => {subscribed.current = false}}, [])
+
+    const createMeeting = async () => {
+        subscribed.current = true;
+        const dateAndTime = new Date(date.current.month + " " + date.current.day + ", " + date.current.year + " " +
+        time.current.hours + ":" + time.current.minutes + ":00 " + time.current.timeOfDay)
+        console.log(dateAndTime)
+        const requestOptionsMeeting = {
+            method: 'POST',
+            body: JSON.stringify(
+              { 
+                  meeting_name: meetingName.current,
+                  meeting_date: dateAndTime,
+                  semester: semester.current,
+                  company_id: company.current.id
+              }
+            )
+        };
+        const res = await fetch('/api/meeting/create', requestOptionsMeeting)
+        const result = await res.json()
+        if (subscribed.current) setCode(result["code"])
+        subscribed.current = false;
+    }
+
+    const handleChange = () => {
+        resetAllErrors()
+        if (meetingName.current && semester.current && date.current.day && time.current.hours) {
+            if (companyMeeting && !company.current.name) {
+                // Company meeting was selected but no companies were found
+                setCompanyError("Must select a valid company or have the meeting not be associated with a company.")
+            } else {
+                createMeeting()
+                setShowNotif(true)
```

Comment:
sorry did not catch this in the dev pr - can you delete this line, then after line 62 in the createMeeting function, put `if(subscribed.current) setShowNotif(true)`